code/modules/species/shadekin/crew_shadekin_abilities.dm

```diff
@@ -0,0 +1,132 @@
+/datum/power/crew_shadekin
+
+/mob/living/carbon/human/is_incorporeal()
+	if(ability_flags & AB_PHASE_SHIFTED) //Shadekin
+		return TRUE
+	return ..()
+
+//////////////////////////
+///  REGENERATE OTHER  ///
+//////////////////////////
+/datum/power/crew_shadekin/crewkin_regenerate_other
+	name = "Regenerate Other (50)"
+	desc = "Spend energy to heal physical wounds in another creature."
+	verbpath = /mob/living/carbon/human/proc/crewkin_regenerate_other
+	ability_icon_state = "tech_biomedaura"
+
+/mob/living/carbon/human/proc/crewkin_regenerate_other()
+	set name = "Regenerate Other (50)"
+	set desc = "Spend energy to heal physical wounds in another creature."
+	set category = "Shadekin"
+
+	var/ability_cost = 50
+
+	if(species.get_species_id() != SPECIES_ID_SHADEKIN)
```

Comment:
Shouldnt this be SPECIES_ID_SHADEKIN_BLACK_EYED?

Reply:
Can do either. What happened was that I made the base Ability use the Species_ID, and didn't change that when I got the weaker abilities to work. Will be changed momentarily.
...
After testing it showed that I had to change it to use the "get_exact_species_id" proc.